### PR TITLE
tasks.c:2067:11: warning: this condition has identical branches [-Wduplicated-branches]

### PR DESCRIPTION
**Pull request by pierrenoel-bouteville-act**
Description
-----------
Replace duplicated if branch because empty by a comment to avoid warning/error with option GCC -Wduplicated-branches

Test Steps
-----------
Build file with -Wduplicated-branches and -Werror with GCC

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tasks.c [hunk 1]
```diff
@@ -2064,15 +2064,12 @@ void vTaskStartScheduler( void )
...
-        else
-        {
-            /* Should only reach here if a task calls xTaskEndScheduler(). */
-        }
+        xPortStartScheduler();
+        /* if just above call of xPortStartScheduler not return pdFALSE
+         *      Should not reach here as if the scheduler is running the
+         *      function will not return.
+         * otherwise
+         *      Should only reach here if a task calls xTaskEndScheduler(). */
```
Comment (aggarg) [inline tasks.c:2072]:
```suggestion
        /* In most cases, xPortStartScheduler() will not return. If it
         * returns pdTRUE then there was not enough heap memory available
         * to create either the Idle or the Timer task. If it returned
         * pdFALSE, then the application called xTaskEndScheduler().
         * Most ports don't implement xTaskEndScheduler() as there is
         * nothing to return to. */

```

Comment (pierrenoel-bouteville-act) [reply tasks.c]: Done

**Comment by pierrenoel-bouteville-act** [comment]
Sorry But it don't understand why formatting failed ?

**Comment by aggarg** [comment]
Please apply the following patch - [formatting.zip](https://github.com/FreeRTOS/FreeRTOS-Kernel/files/8426850/formatting.zip)

**Review by aggarg — APPROVED** [review]

**Review by archigup — APPROVED** [review]

**Pull request merged**